packages/synthetic-chain/src/lib/vat-status.js

```diff
@@ -118,7 +124,8 @@ export const getDetailsMatchingVats = async vatName => {
     const source = vatInfo.source();
     // @ts-expect-error cast
     const { incarnation } = vatInfo.currentSpan();
-    infos.push({ vatName: name, vatID, incarnation, ...source });
+    const terminated = vatInfo.terminated();
+    infos.push({ vatName: name, vatID, incarnation, ...source, terminated });
```

Comment:
I see this had to be implemented in two places. Please refactor `getDetailsMatchingVats` using `getVatDetails`. 

While we're at it, `getDetailsMatchingVats` should also take a `vatSubstring` or something instead of a `vatName` parameter

Reply:
done